keps/sig-auth/3130-kms-observability/kep.yaml

```diff
@@ -14,3 +14,9 @@ approvers:
   - "@smarterclayton"
 stage: alpha
```

Comment:
stable

Reply:
This needs to be the same as https://github.com/kubernetes/enhancements/pull/3133/files#diff-3c134424e5ca25c4b9b0c73ea74c09b970b5f1818809d744c5d585bd3ba00d2bR15. Should we change to stable and update PRR when we make it GA in 1.25?

Reply:
I thought we were just making this whole KEP start at GA / stable (because it is a minor change that should be backwards incompatible).  We have the pointer field+feature gate as a temporary safe guard in case anyone has issues with it.

Reply:
Makes sense! changed it to stable.

Reply:
> I thought we were just making this whole KEP start at GA / stable (because it is a minor change that should be backwards incompatible). 


@enj Do you mean it should be `backwards compatible`?

Reply:
> > I thought we were just making this whole KEP start at GA / stable (because it is a minor change that should be backwards incompatible).
> 
> @enj Do you mean it should be `backwards compatible`?

Yes, typo on my part.